### PR TITLE
making bitwise_not a unop

### PR DESCRIPTION
**Pull request by aethos**
super quick fix

### CODE REVIEW

**Comment by aethos** [comment]
_merge me_

File: src/ast.mli [hunk 1]
```diff
@@ -6,7 +6,7 @@ type bin_op =
   | Leq | Greater | Geq | And
   | Or
 
-type u_op = Not | Neg
+type u_op = Not | Neg | Bit_Not
```
Comment (vlad-scherbich) [inline src/ast.mli:10]: You should also do this:
- move binary BIT ops to `type bin_op`
- kill `type bit_op` altogether
- remove `| Bitop of int * bit_op * int` from `expr`

Comment (aethos) [reply src/ast.mli:10]: oh wait you're right my b

**Comment by aethos** [comment]
_merge me_

File: src/parser.mly [hunk 2]
```diff
@@ -249,10 +249,10 @@ expr:
   | expr LOGIC_GEQ      expr                        { Binop($1, Geq, $3) }
   | expr LOGIC_AND      expr                        { Binop($1, And, $3) }
   | expr LOGIC_OR       expr                        { Binop($1, Or, $3) }
-  | expr BITWISE_AND    expr                        { Binop($1, Bit_And, $3)}
-  | expr BITWISE_OR     expr                        { Binop($1, Bit_Or, $3)}
-  | expr BITWISE_XOR    expr                        { Binop($1, Bit_Xor, $3)}
-  | expr BITWISE_NOT    expr                        { Binop($1, Bit_Not, $3)}
+  | expr BITWISE_AND    expr                        { Binop($1, Bit_And, $3) }
```
Comment (vlad-scherbich) [inline src/parser.mly:252]: +1 for space before `}`

**Review by vlad-scherbich — APPROVED** [review]

**Pull request merged**